app/javascript/src/components/Dashboard/Notes/Card/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import * as dayjs from "dayjs";
+
+import { Card as NoteCard } from "./Card";
+import { DEFAULT_USER_VALUES } from "./constants";
+
+const Card = ({
+  note: { id, created_at, title, description },
+  setNoteIdToDelete,
+}) => {
+  const handleDelete = () => setNoteIdToDelete(id);
+
+  const createdTime = dayjs(created_at);
+  const getTimeDiffInHours = dayjs().hour() - createdTime.hour();
+
+  return (
+    <NoteCard>
+      <NoteCard.Header title={title} handleDelete={handleDelete} />
+      <NoteCard.Body description={description} />
```

Comment:
Let's pass `description` as `children` here. That would provide more flexibility to the component.